strr-web/interfaces/registration-i.ts

```diff
@@ -9,31 +9,32 @@ export interface RegistrationAddressI {
 }
 
 export interface RegistrationI {
-  id: number,
-  registration_number?: string,
+  id: number
+  registration_number?: string
   invoices: {
-    'invoice_id': number,
-    'payment_account': string,
-    'payment_completion_date': string,
-    'payment_status_code': string,
+    'invoice_id': number
+    'payment_account': string
+    'payment_completion_date': string
+    'payment_status_code': string
     'registration_id': number
   }[],
-  listingDetails: { url: string }[],
-  primaryContact: ContactI,
-  secondaryContact: ContactI | null,
-  principalResidence: PrincipalResidenceI,
-  sbc_account_id: number,
-  status: string,
-  submissionDate: string,
-  unitAddress: RegistrationAddressI,
+  listingDetails: { url: string }[]
+  primaryContact: ContactI
+  secondaryContact: ContactI | null
+  principalResidence: PrincipalResidenceI
+  propertyManager: PropertyManagerI
```

Comment:
Same as above, propertyManager should be optional?

Reply:
Updated